'r' is uninitialized in distrib/ulibc/src/stdlib/div.c

This could just be an issue with my compiler but I figured I'd make an issue anyways.
When running 'make', everything builds fine except for div.c. It fails with the following error:
```
src/stdlib/div.c: In function ‘div’:
src/stdlib/div.c:10:10: error: ‘r’ is used uninitialized in this function [-Werror=uninitialized]
   return r;
          ^
cc1: all warnings being treated as errors
```
I am using Debian 10 with Linux 4.4.0 and GCC 8.3.0 from the default repositories. Any help would be appreciated because this looks like it could be a very fun OS to play with.

--- a/distrib/ulibc/src/stdlib/div.c
+++ b/distrib/ulibc/src/stdlib/div.c
@@ -5,7 +5,7 @@
 
 div_t div(int numer, int denom)
 {
-  div_t r;
+  div_t r = {0};
   assert(0 && "not implemented yet.");
   return r;
 }

--- a/distrib/ulibc/src/stdlib/ldiv.c
+++ b/distrib/ulibc/src/stdlib/ldiv.c
@@ -5,7 +5,7 @@
 
 ldiv_t ldiv(long int numer, long int denom)
 {
-  ldiv_t r;
+  ldiv_t r = {0};
   assert(0 && "not implemented yet.");
   return r;
 }

